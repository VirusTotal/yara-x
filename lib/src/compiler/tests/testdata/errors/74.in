global foo rule test { condition: true }