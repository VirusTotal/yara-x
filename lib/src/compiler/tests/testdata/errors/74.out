error: syntax error
 --> line:1:8
  |
1 | global foo rule test { condition: true }
  |        ^ expected `private` or `rule`
  |